changelog/554.misc.rst

```diff
@@ -0,0 +1 @@
+Update contributing documentation to be more explanatory.
```

Comment:
IDK if it's just me, but I feel like "explanatory" isn't the best word to use here

Reply:
This one may be worth removing.